src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class Rectangle extends Figure {
```

Comment:
Consider implementing an interface instead of extending a class. This will make your classes more flexible and allow them to implement multiple behaviors. For example, you could have an interface 'Drawable' with a method 'draw()', and another interface 'AreaCalculable' with a method 'getArea()'.